### ISSUE TITLE
Wobble Goal arm needs a stop mechanism

### ISSUE THREAD

**Issue by ColumbusFTCsoftware** [Bartacus88/FtcRobotController#6]
This still needs some system design - use a touch sensor or use an encoder?   In either case, the controls will need to be developed after a design is agreed to.

**Comment by Bartacus88** [comment]
In 1/5 practice a short automatic program was tested to lift the wobble goal arm and ensure it could hold the weight of a wobble goal with several rings.

**Comment by Bartacus88** [comment]
In 1/7 practice we worked on using LinearActuator.java with our Drive class to run the wobble goal arm this appears to have worked.
See: https://github.com/Bartacus88/FtcRobotController/blob/JK_20_21_Initial_Test/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Driver_90deg_Rotation_JS_Swapped_20_21.java

**Comment by Bartacus88** [comment]
On the 1/9 practice the team tried our code for the wobble arm from the 1/7 practice. The code appears to work and do what we expected. When capturing the goal we started with a 40deg angle and found that was too low to accommodate rings that may be on the wobble. We increased this to 75deg to accommodate the rings. We also found that 160deg was a good transport angle to carry the wobble out of the game field. To place the wobble outside the game field we needed an angle between 80deg and 110deg. To this end gampad1.x = 160deg, gamepad1.b = 80deg, gamepad1.left_bumper = 110 deg.

We also discussed the possibility of using up/down commands rather than direct commands. We discussed moving the wobble arm at 45deg/sec. To this end gamepad2.a = raise wobble arm, gampad2.b = lower wobble arm.
